js/data.js

```diff
@@ -0,0 +1,125 @@
+const projects = [
+  {
+    id: 'project1',
+    title: 'Multi-Post Strories',
+    screenshot: './images/modal-image.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    technologies: [
+      'css', 'html', 'bootstrap', 'ruby',
+    ],
+  },
+  {
+    id: 'project2',
+    title: 'Multi-Post Strories',
+    screenshot: './images/modal-image.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    technologies: [
+      'css', 'html', 'bootstrap', 'ruby',
+    ],
+  },
+  {
+    id: 'project3',
+    title: 'Multi-Post Strories',
+    screenshot: './images/modal-image.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    technologies: [
+      'css', 'html', 'bootstrap', 'ruby',
+    ],
+  },
+  {
+    id: 'project4',
+    title: 'Multi-Post Strories',
+    screenshot: './images/modal-image.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    technologies: [
+      'css', 'html', 'bootstrap', 'ruby',
+    ],
+  },
+
+];
```

Comment:
### Hey Team

As the previous reviewer asked you, please make the necessary changes in the objects of your project's array so that they have a link to the live version and a link to the project source code. It's not important nor expected that you have the proper links right now 😉 The idea is for you to have the proper data structure ready for when you do! You can add your project repo link and your GitHub profile link for the moment 👀

```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    live_link: "link_to_live_version.com",
    source_link: "link_to_source_version.com",
}]
```

Reply:
![image](https://user-images.githubusercontent.com/22321030/151229888-96721d1e-61ad-4dfd-8090-76ec2aa420d6.png)
I have already done so, you could check the code in the branch add-contact-form-validation.

Reply:
also, you can crosscheck in the github pages it is there already

Reply:
@mnekx please be sure that the changes that are requested are implemented and pushed into your active PR branch. I use your PR branch to see if you have made or not the required changes so sadly when I reviewed your project, the changes were not there. Don't forget to add them to this **(add-project-modal)** branch before you ask for another review so that the new reviewer can see your changes 🤓

Reply:
done, could you check? please

Reply:
Addressed 🔥 Go ahead and ask for another review!